charts/flyte-core/values-gcp.yaml

```diff
@@ -199,7 +199,7 @@ db:
       username: flyteadmin
       host: "{{ .Values.userSettings.dbHost }}"
       # -- Create a DB called datacatalog (OR change the name here)
-      dbname: flyteadmin
+      dbname: datacatalog
```

Comment:
Can we also add a step here on this doc https://docs.flyte.org/en/latest/deployment/gcp/index.html#deployment-gcp to create the datacatalog DB